README.md

```diff
@@ -66,6 +66,27 @@ dxscanner [path]
 
 dxs [path]
 ```
+## Flags Usage 🏳️
+**Use flags to let DX Scanner do what you want!**
+
+Insert your auth code if your repo is private.
+```
+dxs -a=AUTH_CODE
```

Comment:
`dxs` is not the default command that we want to "promote" in a readme. The better one is `dx-scanner`